src/main/java/peko/Peko.java

```diff
@@ -0,0 +1,30 @@
+package peko;
+
+import java.io.*;
```

Comment:
Imported classes should be listed explicitly, consider changing the wildcard import